tensorboard/plugins/pr_curve/BUILD

```diff
@@ -0,0 +1,63 @@
+# Description:
+# TensorBoard plugin for precision-recall curves.
+
+package(default_visibility = ["//tensorboard:internal"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+# The public Python api.
+py_library(
+  name = "api",
```

Comment:
Done. I just ran the buildifier command directly instead of using a bazel [] command.